src/dev/src/app/datagrid/detail/detail.html

```diff
@@ -0,0 +1,60 @@
+<!--
+  ~ Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<h2>Detail Pane</h2>
+
+<style>
+clr-datagrid ::ng-deep .datagrid-detail-body {
+  padding: 0 1rem;
+}
+</style>
+
+<clr-datagrid [(clrDgSelected)]="selection">
+  <clr-dg-column [clrDgField]="'id'">User ID</clr-dg-column>
```

Comment:
Hmm should there be some kind of input this column will be used for detail pane? I am thinking of questions like what if the user reorders this column, etc?

Reply:
Good question. This somewhat implicit that the first column is going to be used as the single column during the detail pane being open. It doesn't matter if it was hidden or reordered right now, the first column is displayed. This was part of some discussion earlier, and though we could have an input to define which column to show, the argument was basically that the first column should remain the displayed column because you should put an important column first.